src/formats/dng/dng-read-defines.ts

```diff
@@ -0,0 +1,45 @@
+import {DefinesCreator} from '../../defines/defines-creator';
+import {MagickFormat} from '../../magick-format';
+import {DngOutputColor} from './dng-output-color';
+import {IDefine} from '../../defines/define';
+
+export class DngReadDefines extends DefinesCreator {
+
+  constructor () {
+    super(MagickFormat.Dng);
+  }
+
+  disableAutoBrightness?: boolean;
+
+  outputColor?: DngOutputColor;
+
+  useAutoWhitebalance?: boolean;
+
+  useCameraWhitebalance?: boolean;
+
+  getDefines (): IDefine[] {
+    const defines: IDefine[] = [];
+
+    if (this.hasValue(this.disableAutoBrightness)) {
+      defines.push(this.createDefine('no_auto_bright', this.disableAutoBrightness as boolean));
```

Comment:
Please use `no-auto-bright` here instead and also change the other defines. This is the legacy name, and it seems I also did not change this in Magick.NET yet.

Reply:
I updated name for define

Reply:
Could you also replace `_` with `-` in the other defines?

Reply:
Sure, I changed them.